packages/nibijs/src/chain/types.ts

```diff
@@ -1,4 +1,6 @@
-import { Coin } from "@cosmjs/proto-signing"
+import { Coin, coin as newCoin, coins as newCoins } from "@cosmjs/proto-signing"
+export { newCoin, newCoins }
```

Comment:
Don't export third party packages
Prefer to leave coin and coins without alias'ing their names (There's no name conflicts here)

Reply:
FIxed